recipe-server/normandy/settings.py

```diff
@@ -351,6 +351,8 @@ def RAVEN_CONFIG(self):
     AWS_SECRET_ACCESS_KEY = values.Value()
     AWS_STORAGE_BUCKET_NAME = values.Value()
 
+    GITHUB_URL = values.Value('https://github.com/mozilla/normandy')
```

Comment:
Will this ever be used outside of the `ArgumentEditorMissingError` component? I'm leaning towards just setting this as the default prop versus adding it to the service API call. That way, the link href won't ever be null, too.